src/amo/components/Addon/index.js

```diff
@@ -331,6 +333,12 @@ export class AddonBase extends React.Component {
                   label={this.getFeaturedText(addonType)}
                 />
               ) : null}
+              {isRestartRequired ? (
```

Comment:
I like the idea of a helper, but I do not really like having a state that is 1-1 with the API response. Can we change that?

I could add this helper to ship this feature/fix, but I would like to see a clear separation between app state and API responses. Can I open an issue for that?